### PR TITLE
add a key feature for 5.3 release note

### PR DESCRIPTION
**Pull request by gengliqi**
Signed-off-by: gengliqi <gengliqiii@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a key feature for 5.3 release note.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-5.3.0.md [hunk 1]
```diff
@@ -271,6 +272,7 @@ TiDB 在遥测中新增收集 TEMPORARY TABLE 功能的开启情况。收集的
     - 优化 raft client 错误日志的收集 [#10944](https://github.com/tikv/tikv/pull/10944)
     - 优化日志线程以避免其成为性能瓶颈 [#10841](https://github.com/tikv/tikv/issues/10841)
     - 添加更多的写入查询统计类型 [#10507](https://github.com/tikv/tikv/issues/10507)
+    - 降低写入延迟，从 Raftstore 线程池中分离出 IO 线程池(默认不开启)。具体调优详见 [TiKV 线程池性能调优](/tune-tikv-thread-performance.md) [#10289](https://github.com/tikv/tikv/pull/10289)
```
Comment (qiancai) [inline releases/release-5.3.0.md:275]:
```suggestion
    - 降低写入延迟，从 Raftstore 线程池中分离出 IO 线程池（默认不开启）。具体调优操作，请参考 [TiKV 线程池性能调优](/tune-tikv-thread-performance.md) [#10540](https://github.com/tikv/tikv/issues/10540)
```

File: releases/release-5.3.0.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ TiDB 版本：5.3.0
 + 支持“一键”保存和恢复现场问题的相关信息，提升查询计划问题诊断的效率
 + 支持持续性能分析 (Continuous Profiling) 实验特性，提高数据库性能的可观测性
 + 持续优化存储和计算引擎，提升系统性能和稳定性
++ 降低 TiKV 写入延迟，从 Raftstore 线程池中分离出 IO 线程池(默认不开启)。具体调优详见 [TiKV 线程池性能调优](/tune-tikv-thread-performance.md)
```
Comment (qiancai) [inline releases/release-5.3.0.md:22]:
```suggestion
+ 降低 TiKV 写入延迟，从 Raftstore 线程池中分离出 IO 线程池（默认不开启）
```

**Review by qiancai — COMMENTED** [review]
@gengliqi I've completed the review and made two comments. PTAL.

**Comment by easonn7** [comment]
LGTM

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**